### DISCUSSION TITLE
Флоу автоматизации

### DISCUSSION THREAD

**Discussion by igsekor** [doka-guide/content#1296] [Идеи] [answered]
Сейчас все экшены работают исключительно параллельно. Предлагается оптимизация с помощью организации порядка запуска экшенов. Когда предыдущий экшен заканчивается успешно, следующий начинает свою работу. Кроме этого, в черновиках не имеет смысла запускать не только превью, но и ряд других экшенов. Поэтому предлагается следующие последовательности:

**Для pull_request**

```
 EditorСonfig
      |
  YaSpeller
      |
    Surge
```

**Для main**

```
 EditorСonfig
      |
  YaSpeller -- Sort Dictionary
      | 
 Update Index
      |
   Netlify
```

**Comment by solarrust** [comment]
@igsekor на сколько я понимаю, ты реализовал идею с последовательными экшенами?

**Comment by nlopin** [comment, marked as answer]
сделано в #1330
